### DISCUSSION TITLE
She'll command to verify tunnel

### DISCUSSION THREAD

**Discussion by sollermun** [Mygod/VPNHotspot#269] [General]
Is there a shell command that I can use to verify that the rndis tunnel is active?

**Comment by Mygod** [comment]
`ip monitor link`?

**Comment by sollermun** [comment]
What is the command I need to use to see if the toggle has been flipped on for rndis0, i.e. that the usb tether is being routed through VPN. To cut to the chase, it would be nice to have a feature for the app that ensures that tether data is always sent over VPN. I do not want any tethered data that does not go over the VPN.

While the toggle is nice, sometimes it will be untoggled, e.g. if the tether connection is restarted. The android VPN kill switch does not seem to extend to tethered data.

**Reply by Mygod** [reply]
You can use the monitor feature. After USB tethering is on, click on the eye icon on the top right.

**Comment by sollermun** [comment]
Thank you!!! You can ignore a similar comment I left on a different post. I appreciate your work!
